documentation/source/user_section/tutorials/spinalcord-segmentation.rst

```diff
@@ -0,0 +1,24 @@
+.. _spinalcord-segmentation:
+
+Spinal cord segmentation for anatomical images
+##############################################
+
+This tutorial demonstrates how to use SCT's command-line scripts to perform spinal cord segmentation. Segmentation in this case means to generate a 3D mask that identifies the spinal cord within anatomical images of the spine. This tutorial compares two different algorithms provided by SCT, and is meant to give you a feel for common usage of these tools on real-world data.
```

Comment:
```suggestion
This tutorial demonstrates how to use SCT's command-line scripts to perform spinal cord segmentation. In image processing, [segmentation](https://en.wikipedia.org/wiki/Image_segmentation) is the process of partitioning an image into different segments (sets of pixels/voxels). In the context of SCT, we generate a 3D mask (solid region overlaying segment of interest) that identifies the spinal cord within anatomical images of the spine. This tutorial compares two different algorithms provided by SCT, and is meant to give you a feel for common usage of these tools on real-world data.
```

Reply:
Thank you for the suggestion! I really appreciate your perspective here -- clarity tweaks are super important for making the documentation accessible. :slightly_smiling_face:

Reply:
No problem! :)